src/Providers/Images/VgmdbAlbumImageProvider.cs

```diff
@@ -14,32 +15,28 @@ namespace Jellyfin.Plugin.Vgmdb.Providers.Images
 {
 	public class VgmdbAlbumImageProvider : IRemoteImageProvider
 	{
-		private readonly IHttpClient _httpClient;
+		private readonly IHttpClientFactory _httpClientFactory;
 		private readonly VgmdbApi _api;
 
-		public VgmdbAlbumImageProvider(IHttpClient httpClient, IJsonSerializer json)
+		public VgmdbAlbumImageProvider(IHttpClientFactory httpClientFactory, IJsonSerializer json)
 		{
-			_httpClient = httpClient;
-			_api = new VgmdbApi(httpClient, json);
+			_httpClientFactory = httpClientFactory;
+			_api = new VgmdbApi(httpClientFactory, json);
 		}
 
 		public string Name => "VGMdb";
 
-		public Task<HttpResponseInfo> GetImageResponse(string url, CancellationToken cancellationToken)
+		public Task<HttpResponseMessage> GetImageResponse(string url, CancellationToken cancellationToken)
 		{
-			return _httpClient.GetResponse(new HttpRequestOptions
-			{
-				Url = url,
-				CancellationToken = cancellationToken
-			});
+			return _httpClientFactory.CreateClient().GetAsync(url, cancellationToken);
```

Comment:
```suggestion
			return _httpClientFactory.CreateClient(NamedClient.Default).GetAsync(url, cancellationToken);
```
From `MediaBrowser.Common.Net`
Adds compression and proper UserAgent.
(Use in all `.CreateClient`)

Reply:
I guess that's new since the PRs I used as reference.
Am I supposed to always `.CreateClient()` in all functions that use it?

Reply:
Yes, `GetClient` should be used only in the function where the `HttpClient` is needed